src/main/java/sk/tomsik68/mclauncher/impl/login/yggdrasil/YDLoginService.java

```diff
@@ -22,33 +23,52 @@
 
 public final class YDLoginService implements ILoginService {
     public static UUID clientToken = UUID.randomUUID();
-    private static String PASSWORD_LOGIN_URL = "https://authserver.mojang.com/authenticate";
-    private static String SESSION_LOGIN_URL = "https://authserver.mojang.com/refresh";
-    private static String SESSION_LOGOUT_URL = "https://authserver.mojang.com/invalidate";
+    private final String PASSWORD_LOGIN_URL;
```

Comment:
Please rename these three fields to `lowerCamelCase` since they're no longer constants and you're good to go!